Amazons3 guide (#472)
wso2/ballerina-integrator: docs/content/src/learn/tutorials/saas-integrations/amazons3/amazon-s3-bucket-service/src/integration-with-amazon-s3-bucket/Module.md

Guide on Amazon S3 Bucket Service 

# Guide Overview

## About

Ballerina is an open-source programming language that empowers developers to integrate their system easily with the 

> support of connectors. In this guide, we are mainly focusing on connecting to the Amazon Simple Storage Service API to create, store,download, and use data with other services.  

Reviewer: ```suggestion
support of connectors. In this guide, we are mainly focusing on connecting to the Amazon Simple Storage Service API to create, store, download, and use data with other services.  
```

Author: Fixed

The `wso2/amazons3` module allows you to perform the following operations.

This example explains how to use the S3 client to connect with the Amazon S3 instance and to create a Amazon S3 bucket.

> You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) Github organization.

Reviewer: ```suggestion
You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) GitHub organization.
```

Author: Fixed

Author: Fixed

## What you'll build

> This application connects with the Amazon S3 API and create a new bucket on Amazon S3 instance with the provided name, get the available buckets and delete the specified bucket.

Reviewer: ```suggestion
This application connects with the Amazon S3 API and creates a new bucket on Amazon S3 instance with the provided name. It gets the available buckets and deletes the specified bucket.
```

Author: Fixed
